### ISSUE TITLE
Adding PDF/A generation notebook

### ISSUE THREAD

**Issue by felixdittrich92** [mindee/notebooks#2]
[notebook](https://colab.research.google.com/drive/1_gP3yVQdHpw43LAvGGaWsuKGKj775xFT?usp=sharing)

**Comment by fg-mindee** [comment]
Sorry about the late replies, we've had a lot of things to take care of :sweat_smile: 
This looks quite nice! As a user looking for examples, I would still missing one thing though: what does it mean to have a searchable-PDF / how do I use this searchable PDF.

So I can think of two options:
- something can be done with it in Python, and thus we add a cell to search the PDF / showcase the features it enables
- not easy to do in Python, so we add references to articles / resources about PDF/A and what you can do with it

**Comment by felixdittrich92** [comment]
@fg-mindee 
done 
i have added two new sections:
```
# 4. And now ? What can we do with this results ?
### you can open the saved pdfs with any PDF Viewer and type some words you are searching for in the document.
### matches will be highlighted in the textlayer. 
### or you use Python to search the words in the textlayer. (which is demonstrated in the next section)
```
```
# search specific words in the pdf and print all matches

import PyPDF2
import re

pattern = input("Enter string pattern to search: ")
file_name = input("Enter file path and name: ")

reader = PyPDF2.PdfFileReader(file_name)
num_pages = reader.getNumPages()

for i in range(0, num_pages):
    page = reader.getPage(i)
    text = page.extractText()

    for match in re.finditer(pattern, text):
        print(f'Page no: {i} | Match: {match}')
```
Output:
Page no: 0 | Match: <re.Match object; span=(845, 852), match='Beitrag'>
Page no: 0 | Match: <re.Match object; span=(1146, 1153), match='Beitrag'>
Page no: 0 | Match: <re.Match object; span=(1184, 1191), match='Beitrag'>
Page no: 0 | Match: <re.Match object; span=(1268, 1275), match='Beitrag'>
Page no: 0 | Match: <re.Match object; span=(1312, 1319), match='Beitrag'>

```
# 5. further resources
### https://en.wikipedia.org/wiki/PDF/A
### https://askanydifference.com/difference-between-pdf-a-and-pdf/

### Happy Coding :)
```

wdyt ?

**Comment by fg-mindee** [comment]
Getting back to this:
- first, I forgot to tell you, but it would be nice to better organize the title hierarchy (we use `#` for notebook title, `##` for sections, `###` for subsections, etc.)
- a title is better if it's not too long "And now ? What can we do with this results ?" --> "How can I use a PDF/A?"
- you can remove the "Tutorial" from the main title
- also, it's nice to have a sensible file name, I suggest "export_as_pdfa.ipynb"
- let's move all imports up to a single cell
- for the 3 export options, please order them : 1 subsection title + corresponding cell, then move to the next
- Make sure every sentence starts with an uppercase, ends up with a dot, and that we have proper grammar in it :) 
- having the user input something is nice, but only looking at the notebook, it's not as clear as if we use the generated pdfA + a predefined string pattern
- for the "further resources" --> "To go further"

Let me know if that's alright with you :+1:

**Comment by felixdittrich92** [comment]
@fg-mindee 
sounds really good :+1: 
Can we write it down as "rule" for notebooks ? :smile:

**Comment by felixdittrich92** [comment]
@fg-mindee 
please inspect again but i think now it is a good fit. I have also improved the notebook flow the cells can now run each behind the other with showing the results and also a bit more understandable with visualization
[https://colab.research.google.com/drive/1bn0OcaofFwrfKxgRFXZTi6JyViCCCtjO?usp=sharing](https://colab.research.google.com/drive/1bn0OcaofFwrfKxgRFXZTi6JyViCCCtjO?usp=sharing)

**Comment by fg-mindee** [comment]
> @fg-mindee sounds really good +1 Can we write it down as "rule" for notebooks ? smile

Great idea, we'll have to update the instructions in the README then!

**Comment by fg-mindee** [comment]
Closed by #3

**Comment by fg-mindee** [comment]
@felixdittrich92 Feel free to open a PR on docTR to add the entry in https://github.com/mindee/doctr/blob/main/notebooks/README.md :+1:
